src/sentry/api/serializers/models/filechange.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register
+from sentry.models import CommitFileChange
+from sentry.api.serializers.models.release import get_users_for_commits
+
+
+@register(CommitFileChange)
+class CommitFileChangeSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+        commits = [f.commit for f in item_list]
```

Comment:
This is an O(n) query to fetch the `Commit` rows.

I'd change this to:

```python
commits = Commit.objects.filter(id__in=[f.commit_id for f in item_list]).select_related('author')
```

or somethign akin to this to bring it back to O(1)